exonum/src/runtime/mod.rs

```diff
@@ -168,15 +255,19 @@ impl Caller {
     }
 }
 
+/// Provides the current state of the blockchain and caller information for the transaction
+/// which being executed.
 #[derive(Debug)]
 pub struct ExecutionContext<'a> {
+    /// Reference of the actual state of the blockchain with new uncommitted changes.
```

Comment:
The current state of the blockchain. It includes the new, not-yet-committed, changes to the database made by the previous transactions already executed in this block.